nixos/modules/programs/ydotool.nix

```diff
@@ -0,0 +1,33 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.programs.ydotool;
+in
+
+{
+  ###### interface
```

Comment:
We don't need those comments. They are rather useless and an artifact of the past.